ADAL/src/ADAuthenticationError.m

```diff
@@ -333,6 +333,7 @@ + (NSString*)stringForADErrorCode:(ADErrorCode)code
             AD_ERROR_CODE_ENUM_CASE(AD_ERROR_TOKENBROKER_RESPONSE_NOT_RECEIVED);
             AD_ERROR_CODE_ENUM_CASE(AD_ERROR_TOKENBROKER_FAILED_TO_CREATE_KEY);
             AD_ERROR_CODE_ENUM_CASE(AD_ERROR_TOKENBROKER_DECRYPTION_FAILED);
+            AD_ERROR_CODE_ENUM_CASE(AD_ERROR_DEVELOPER_JSON_DATA_STRUCTURE_MALFORMED);
```

Comment:
I think the JSON is not provided by developer, right? :)

Reply:
It should be provided by the MAM SDK, so that's correct, I'm not sure which other bucket it fits into though.

Reply:
Hmm, we could just use AD_ERROR_UNEXPECTED for this code, as we'd only hit this if the app unexpectedly modifies NSUserDefaults

Reply:
Talked to Olga, she agreed that we can use AD_ERROR_UNEXPECTED :)